### PR TITLE
Support setting host attributes via the command line

### PR DESCRIPTION
**Pull request by lmars**

### CODE REVIEW

File: lorne.go [hunk 1]
```diff
@@ -27,12 +27,34 @@ var Attempts = attempt.Strategy{
...
+}
+
 func main() {
 	hostname, _ := os.Hostname()
 	externalAddr := flag.String("external", "", "external IP of host")
 	configFile := flag.String("config", "", "configuration file")
 	manifestFile := flag.String("manifest", "", "manifest file")
 	hostID := flag.String("id", hostname, "host id")
+	attributes := make(map[string]string)
+	flag.Var(&AttributeFlag{Attributes: attributes}, "attribute", "key-value pairs to set as attributes")
```
Comment (titanous) [inline lorne.go]: `"key=value pair to add as an attribute"`

File: lorne.go [hunk 2]
```diff
@@ -27,12 +27,34 @@ var Attempts = attempt.Strategy{
 	Delay: 200 * time.Millisecond,
 }
 
+// A command line flag to accumulate multiple key-value pairs into Attributes,
+// e.g. flynn-host -attribute foo=bar -attribute bar=foo
+type AttributeFlag struct {
+	Attributes map[string]string
```
Comment (titanous) [inline lorne.go]: You should be able to do `type AttributeFlag map[string]string`.

Comment (lmars) [inline lorne.go]: ah ok, updated: 2837362

File: lorne.go (hunk 1 above)
Comment (lmars) [inline lorne.go]: updated: 2837362

**Pull request merged**